Source/Classes/PINRemoteImageManager.m

```diff
@@ -784,13 +786,62 @@ - (void)downloadImageWithURL:(NSURL *)url
             NSError *remoteImageError = error;
             PINImage *image = nil;
             id alternativeRepresentation = nil;
-             
+            __block NSNumber *maxAge = nil;
             if (remoteImageError == nil) {
-                 //stores the object in the caches
-                 [self materializeAndCacheObject:data cacheInDisk:data additionalCost:0 url:url key:key options:options outImage:&image outAltRep:&alternativeRepresentation];
+                if (_isTtlCache && !(options & PINRemoteImageManagerDownloadOptionsIgnoreCacheControlHeaders)) {
```

Comment:
Believe it or not, this bitwise-AND should not be inverted / handled as a BOOL as it can fail on specifically 32-bit ARM! Our team had some nasty issues with this last Fall.

If other parts of the file have this bug as well, we should introduce a macro similar to ASInterfaceStateIncludes() or  ASCellLayoutModeIncludes() to resolve.